### DISCUSSION TITLE
Towing system

### DISCUSSION THREAD

**Discussion by SOF-Ice** [Vdauphin/HeartsAndMinds#1437] [Question and answer] [answered]
Hi Vdauphin,

I'm actually making a new H&M mission but i have a problem

I would like to tow planes to park them in air-shed. but the towing system of H&M dont want to let me towing them.
And 
planes come from two mods which are not supported by lesh towing.

My question is:

Can i make a vehicle able to tow my plane with the H&M towing system ?

**Comment by Vdauphin** [comment, marked as answer]
Hello,

> Can i make a vehicle able to tow my plane with the H&M towing system ?

Yes see https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/mission.sqf#L539-L552
Remove plane where you don't want

Cheers

**Reply by SOF-Ice** [reply]
Thanks for help ! Cheers

**Reply by margotwilliamm-spec** [reply]
Yes, it is possible to make a vehicle capable of towing planes with the H&M towing system, but it usually requires some configuration or mod support. Since your planes come from two different mods that are not supported by Lesh Towing, the default H&M system will not recognize them as towable by default.

To fix this, you generally have three options:

Edit the plane vehicle configs – You can add or adjust towing and attachment points in the plane’s config files so H&M recognizes them as towable vehicles. This allows the towing system to detect the aircraft correctly.

Create or modify a dedicated aircraft tug – The best solution is to use or create a special aircraft tug vehicle designed for planes. Many communities use aircraft tug-style vehicles that can pull planes by aligning with the nose wheel and using custom attach points.

Request or add compatibility – If possible, ask the mod authors to add H&M towing compatibility, or manually add the required class names of your planes into the H&M towing script so they are treated as supported vehicles.

In real-world operations, specialized towing vehicles are always used to safely move aircraft into hangars and air-sheds. The same principle applies in-game: using the correct tug vehicle with proper towing support will give you the best and most realistic results—just like professional towing services such Sydney Towing Trucks handle specialized towing needs in real life: https://sydneytowingtrucks.com.au/
